releasenotes/notes/munir-deprecate-rc-http-configs-206f6c60ff383c05.yaml

```diff
@@ -0,0 +1,11 @@
+---
+deprecations:
+  - |
+    tracing: Internalizes the ``ddtrace.settings.config`` module and deprecates the following ``ddtrace.config`` attributes:
```

Comment:
Can we add the alternative options here? Envars? Or at least say to use their envar counterparts?